### PR TITLE
Added convensions, resources and links

### LINKED ISSUE #3: Write about conventions used in this book
Mention all the conventions used in this book so that it will easy to understand for reader. Probably we could complete this after chapter1.

### PR DESCRIPTION
**Pull request by KhushbuB**
Fixed #3 
Added Convensions and resources to the book.
Signed-off-by: KhushbuB <khushbu.borole@gmail.com>

### CODE REVIEW

**Comment by akshaygin** [comment]
LGTM

**Comment by son-vyas** [comment]
I think the written content still needs an attention.

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Now a days, Automation is the mother of all the inventions, that give rise to al
 
 A Hybrid cloud is the one with the combination of both i.e. private as well as public cloud with the specified specifications.
 
-ManageIQ is an open source cloud management platform by Red Hat. It is written in Ruby with RoR (Ruby on Rails) as platform. ManageIQ have different releases that will be discussed in chapter 1. Fine is a released version of ManageIQ, since then it has supported automation. It also helps integrate Openstack.
+ManageIQ is an open source cloud management platform by Red Hat. It is written in Ruby with RoR (Ruby on Rails) as platform. ManageIQ have different releases that will be discussed in [chapter 1.1](chapter1/intro_to_MIQ.md). Fine is a released version of ManageIQ, since then it has supported automation. It also helps integrate Openstack.
```
Comment (omkar0496) [inline README.md]: I think 'Openstack' word should be 'OpenStack'. Same applies to all the text.

**Review by omkar0496 — CHANGES_REQUESTED** [review]

**Comment by son-vyas** [comment]
I feel like following content is missing:-
1. Need for integration for OpenStack provider in ManageIQ
2. Need for Ansible integration

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The automation helps the globe lead the nation forward and work efficiently. The
 - [**_Chapter 1_**](chapter1/README.md): This contains the introduction of the main constrain of the project i.e. ManageIQ. ManageIQ installation, its appliance console and basic configuration is also discussed.
```
Comment (son-vyas) [inline README.md]: It should be **'constraint'** I guess?

**Comment by son-vyas** [comment]
LGTM

**Comment by akshaygin** [comment]
@KhushbuB  Thank you.

**Pull request merged**